_tests/browser_tests/.eslintrc

```diff
@@ -0,0 +1,10 @@
+env:
+  node: true
+  jasmine: true
+rules:
+  no-unused-expressions: 0
+  no-process-env: 0
+globals:
+  by: true
+  browser: true
```

Comment:
Do the tests have a global variable called `browser`? Otherwise this shouldn't be needed as `browser: true` is already set in the `env` section of the main eslintrc file.